modules/catalog-magento/composables/useProduct.ts

```diff
@@ -99,6 +105,13 @@ export default (product: Ref<ReturnType<typeof ToProduct>>) => {
 
   const productOptions = computed(() => product.value.options ?? [])
 
+  const isOptionsConfigured = computed(() => {
+    if (!isNonEmptyObject(options.value)) {
```

Comment:
This seems redundant. Can this computed just be `computed(() => isNonEmptyObject(options.value))` ?